payments/card/after-payment.md

```diff
@@ -574,7 +576,7 @@ Content-Type: application/json
 
 {
   "state": "Deleted",
-  "tokenType" : "PaymentToken|RecurrenceToken",
+  "tokenType" : "PaymentToken | RecurrenceToken",
```

Comment:
No pipes in examples, please. Explain the possible values in below property tables instead.

```suggestion
  "tokenType" : "PaymentToken",
```